config/initializers/cognito_authenticable.rb

```diff
@@ -0,0 +1,68 @@
+require 'aws-sdk'
+require 'devise/strategies/authenticatable'
+
+module Devise
+  module Strategies
+    class CognitoAuthenticatable < Authenticatable
+      def authenticate!
```

Comment:
this is a little bit hard to read, could you run `bundle exec govuk-lint-ruby config/initializers/cognito_authenticable.rb` to lint this?

Reply:
Done